.ci/openshift-ci/build-root/e2e-test.sh

```diff
@@ -16,33 +16,71 @@ printenv KUBEADMIN_PASSWORD_FILE
 oc get node
 oc config view
 
-# For some reason the creation of a cluster admin and a user authentication mechanism does not allow for immediate
-# login, even though the oc commands don't report any error.
-# This doesn't let us get a token, and XTF won't take the xtf.openshift.admin.kubeconfig and
-# xtf.openshift.master.kubeconfig properties into account when the token is not present.
-# Eventually this turns out to be the condition that prevents the oc binary commands to succeed, because they would
-# use a tmp oc.config file which is not found on the build cluster filesystem.
-# This is why we login with kubeadmin, get a token and use its full set of credentials (username, password and token)
-# both for xtf.openshift.admin.* and xtf.openshift.master.* properties
+export TEST_CLUSTER_URL=$(oc whoami --show-server)
+
 export SYSADMIN_USERNAME=kubeadmin
 export SYSADMIN_PASSWORD=$(cat "${KUBEADMIN_PASSWORD_FILE}")
-export TEST_CLUSTER_URL=$(oc whoami --show-server)
-oc login "${TEST_CLUSTER_URL}" -u "${SYSADMIN_USERNAME}" -p "${SYSADMIN_PASSWORD}" --insecure-skip-tls-verify=true
-export SYSADMIN_TOKEN=$(oc whoami -t)
+
+# note that for some reason it takes a few minutes for this to be loaded by OCP and authentication start working
```

Comment:
@fabiobrz why do we need yet another cluster admin?

Reply:
See https://github.com/Intersmash/intersmash/issues/48 :slightly_smiling_face: